src/model/Guess.java

```diff
@@ -38,6 +38,45 @@ public Guess(String nameIn, String typingIn, String paradigmIn, String levelIn,
      * @param other the guess to compare the current guess to
      * @return a boolean representing equality
      */
+
+    public String getName()
+    {
+        return name;
+    }
+    
+    public String getTyping()
+    {
+        return typing;
+    }
+    
+    public int getParadigmID()
+    {
+        return paradigmID;
+    }
+    
+    public String getParadigmName()
+    {
+        switch(paradigmID)
+        {
+            case 1 -> paradigmName = "PROCEDURAL";
+            case 2 -> paradigmName = "STRUCTURED";
+            case 3 -> paradigmName = "OBJECT ORIENTED";
+            case 4 -> paradigmName = "FUNCTIONAL";
+            case 5 -> paradigmName = "LOGIC";
+        }
+        return paradigmName;
+    }
```

Comment:
Honestly, I would flip this function around to convert from a String to an ID. IDs are less readable and should probably just be used internally for easy comparisons.